lib/bigquery/dataset.js

```diff
@@ -64,6 +64,9 @@ function Dataset(bigQuery, id) {
  *     JSON array of fields, which allows for nested and repeated fields. See
  *     a [Table resource](http://goo.gl/sl8Dmg) for more detailed information.
  * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request
+ * @param {module:bigquery/table} callback.table - The newly created table.
```

Comment:
I'm not sure on the right approach here, as far as if this is `nullable` or not. Technically, the callback will get the `table` argument only in non-error cases, so should `{module:bigquery/table}` be `{?module:bigquery/table}`? The only constant, never-null argument is `callback.apiResponse`.

I don't know if that's just going to make things confusing by being technically correct to that detail. Any thoughts?